### PR TITLE
Allow CRUD to work with non-RDBMS ARs

### PR DESCRIPTION
**Pull request by WinterSilence**
| Q             | A
| ------------- | ---
| Is bugfix?    | yes
| New feature?  | yes
| Breaks BC?    | no
| Tests pass?   | yes
| Fixed issues  | #452

### CODE REVIEW

File: src/generators/crud/Generator.php [hunk 1]
```diff
@@ -25,24 +25,56 @@
...
+    /**
+     * @var string
+     */
     public $controllerClass;
+    
+    /**
+     * @string string[] Allowed widgets for index view
+     */
+    public $indexWidgetTypes = [
+        'grid' => 'yii\grid\GridView',
```
Comment (samdark) [inline src/generators/crud/Generator.php:48]: Haven't we agreed that it should not be part of the same PR?

Comment (WinterSilence) [reply src/generators/crud/Generator.php:48]: @samdark no problem, i just fix problem in php5.4

**Review by schmunk42 — APPROVED** [review]

**Review by samdark — APPROVED** [review]
Looks alright, please add a line to changelog. Thanks.

**Comment by WinterSilence** [comment]
@samdark changelog was updated

**Pull request merged**